vmdb/app/controllers/api_controller/vms.rb

```diff
@@ -60,5 +73,17 @@ def stop_vm(vm, klass)
     rescue => err
       action_result(false, err.to_s)
     end
+
+    def suspend_vm(vm, klass)
+      desc = "#{vm_ident(vm)} suspending"
+      task_id = queue_object_action(vm,
+                                    desc,
+                                    :class_name  => klass.name,
```

Comment:
I just realized the class_name is not needed. You are passing in the vm, so you *have* the class name.  I'l merge for now, and we can clean up in a follow up PR.